app/screens/claim/index.tsx

```diff
@@ -1,21 +1,61 @@
-﻿﻿import React from "react";
-import { Text, View } from "react-native";
+﻿﻿import React, { useEffect } from "react";
+import { Text, View, Platform, Image } from "react-native";
 import styles from "../../styles";
 import EmailClaim from "./EmailClaim";
-import * as DocumentPicker from "expo-document-picker";
 import { useRootStore } from "../../store/rootStore";
 import { observer } from "mobx-react-lite";
 import DateClaim from "./DateClaim";
 import OtherClaim from "./OtherClaim";
 import MobileClaim from "./MobileClaim";
 import SelectClaim from "./SelectClaim";
+import * as ImagePicker from "expo-image-picker";
+import * as DocumentPicker from "expo-document-picker";
+import AsyncStorage from "@react-native-async-storage/async-storage";
 
 const Claim = () => {
+  useEffect(() => {
+    (async () => {
+      if (Platform.OS !== "web") {
+        const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+        if (status !== "granted") {
+          alert("Sorry, we need camera roll permissions to make this work!");
+        }
+      }
+    })();
+  }, []);
+
   const uploadFile = async () => {
```

Comment:
this shouldnt be called upload if you are storing it to local stoage.

Reply:
Change function name UploadFile -> UploadFileFromBrowser